GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTPrioritizer.m

```diff
@@ -29,6 +29,8 @@ @implementation GDTCCTPrioritizer
 + (void)load {
   GDTCCTPrioritizer *prioritizer = [GDTCCTPrioritizer sharedInstance];
   [[GDTCORRegistrar sharedInstance] registerPrioritizer:prioritizer target:kGDTCORTargetCCT];
+  [[GDTCORRegistrar sharedInstance] registerPrioritizer:prioritizer target:kGDTCORTargetFLL];
```

Comment:
Should the file be renamed?

Reply:
I think it's fine--they all use the CCT proto, so CCT is still an ok name imo